### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by darxriggs**
These are some minor code improvements.

### CODE REVIEW

File: src/main/java/org/jenkinsci/plugins/envinject/EnvInjectPasswordWrapper.java [hunk 1]
```diff
@@ -197,7 +196,7 @@ public void makeSensitiveBuildVariables(AbstractBuild build, Set<String> sensiti
     /**
      * Class took from the mask-passwords plugin
      */
-    class EnvInjectPasswordsOutputStream extends LineTransformationOutputStream {
+    static class EnvInjectPasswordsOutputStream extends LineTransformationOutputStream {
```
Comment (oleg-nenashev) [inline src/main/java/org/jenkinsci/plugins/envinject/EnvInjectPasswordWrapper.java]: Makes sense to me though it might cause serialization issues in edge cases

File: src/test/java/org/jenkinsci/plugins/envinject/EnvInjectEvaluatedGroovyScriptTest.java [hunk 2]
```diff
@@ -186,7 +186,7 @@ public void testWorkaroundSecurity86() throws Exception {
         job.addProperty(property);
 
         TestUtils.saveConfigurationAs(jenkins, job, "alice");
-        //Since alice is an admin the script should be approved automagically
+        //Since alice is an admin the script should be approved automatically
```
Comment (oleg-nenashev) [inline src/test/java/org/jenkinsci/plugins/envinject/EnvInjectEvaluatedGroovyScriptTest.java]: It was intended :)
